bin/cli.js

```diff
@@ -83,7 +83,7 @@ function getAssetData(config) {
     assetsRoot = path.dirname(configPaths.configPath);
   }
   debug('Root directory for assets: %s', assetsRoot);
-  Object.keys(assets).forEach(function(key) {
+  Object.keys(assets).forEach(function (key) {
```

Comment:
I see a lot of white-space changes like this, which make it difficult to spot the actual breaking changes. Can you please isolate the auto-generated changes into a standalone commit please? I assume this is caused by `tslint@6`, so perhaps create a standalone commit for bumping up `tslint` version and applying the new code style?